action_access_control_list/controllers/main.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+# © 2017 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/gpl).
+
+import logging
+from odoo.api import Environment
+from odoo.http import request
+from odoo.addons.web.controllers.main import DataSet
+
+
+_logger = logging.getLogger(__name__)
+
+EXCLUDED_MODELS = {'res.users'}
+
+BASE_METHODS = {
```

Comment:
@dufresnedavid I'm not sure what's your interpretation of base method but the list seems incomplete to me. What about `load`, `search_read`, `browse`, `read_group`, `name_create`, `fields_view_get`...

Reply:
@joaoalf right, I need to add the missing methods.